biolink/api/link/endpoints/associations_from.py

```diff
@@ -67,3 +67,23 @@ def get(self, subject, object):
         """
         args = core_parser.parse_args()
         return search_associations(subject=subject, object=object, user_agent=USER_AGENT, **args)
+
+@api.doc(params={'association_type': 'Association type, eg gene_phenotype'})
```

Comment:
will you also support subject.cat and object.cat (to avoid forcing string concatenation)?

Reply:
I added [association_type](https://github.com/monarch-initiative/monarch-app/commit/ff749344cfe912317fab47c1c0c30653dcd61143#diff-afd371b82f73195d2b80e578810731d7) to the monarch solr 6 months ago to help with facet counting, mostly to disambiguate gene-gene associations, this endpoint queries this field directly.  There's a subject_cat, object_cat endpoint already, /association/find/{subject_category}/{object_category}